public/app/features/provisioning/RepositoryStatusPage.tsx

```diff
@@ -52,6 +54,7 @@ const tabInfo: SelectableValue<TabSelection> = [
   { value: TabSelection.Files, label: 'Files', title: 'The raw file list from the repository' },
   { value: TabSelection.Jobs, label: 'Recent events' },
   { value: TabSelection.Export, label: 'Export' },
+  { value: TabSelection.Edit, label: 'Edit' },
```

Comment:
I think we can remove the `Edit` button in the listing page as the button says `Manage` and changing configuration is something that should rarely happen.